apps/renderer/src/assets/main.css

```
@@ -1,33 +1,135 @@
+ * 機械的制約 (constraint over determinism):
+ *   - 生 Tailwind palette (bg-zinc-800 / var(--color-zinc-N) 等) → `gozd/no-raw-tailwind-palette`
+ *   - primitive 直参照 (var(--gray-3) 等) → 同 rule で error
+ *   - 命名は role-based。numbered semantic alias は禁止 (panel / element 等)
+ * ------------------------------------------------------------------------- */
 
-/* 細いオーバーレイスクロールバー */
-::-webkit-scrollbar {
-  width: 6px;
-  height: 6px;
-}
+@theme inline {
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)